### DISCUSSION TITLE
Add Dasel to Hosted Linux and Windows Github Runners

### DISCUSSION THREAD

**Discussion by bramvdklinkenberg** [community/community#12548] [Actions]
I would like to see [Dasel](https://github.com/tomwright/dasel) being added to the hosted linux and windows github runners. 
It is comparable to [jq](https://github.com/stedolan/jq) / [yq](https://github.com/kislyuk/yq), but supports JSON, YAML, TOML, XML and CSV with zero runtime dependencies.
The PUT even allows you to edit your data.

**Comment by ethomson** [comment]
Hi @bramvdklinkenberg - I appreciate the feedback. I'll mention this to the team. But for this particular request, I think that the best place is to open an issue in the [virtual environments](https://github.com/actions/virtual-environments) repository, which is the repository for the images themselves. This tool sounds useful, but the team has to balance the image size with the utility, and they consider each application request carefully.

Thanks much for the feedback.
